docs/modules/ROOT/pages/how-tos/metadata/sboms.adoc

```diff
@@ -9,7 +9,7 @@ You can use an SBOM to better understand the composition of your software. In pa
 
 .Prerequisites
 
-* You must have an application that {ProductName} has successfully built and deployed.
+* You must have a component that {ProductName} has successfully built and tested.
```

Comment:
Nit: The ITS don't need to run, just the build pipeline. So these would just be the build-time tests (which by default should not cause the PipelineRun to fail even if the tests return issues).